src/Types/Types/Scalars/DateTimeType.cs

```diff
@@ -18,6 +18,11 @@ public DateTimeType()
 
         protected override string Serialize(DateTime value)
         {
+            if (value.Kind == DateTimeKind.Utc)
+            {
+                return value.ToString("yyyy-MM-ddTHH\\:mm\\:ss.fffZ");
```

Comment:
We should align the outputs. The else clause is without fff. So, either we include in both serialization formats the milliseconds or in none of the outputs. Can you fix that?